src/audio/base_fw.c

```diff
@@ -111,6 +111,11 @@ static int basefw_config(uint32_t *data_offset, char *data)
 	sche_cfg.sys_tick_source = SOF_SCHEDULE_LL_TIMER;
 	tlv_value_set(tuple, IPC4_SCHEDULER_CONFIGURATION, sizeof(sche_cfg), &sche_cfg);
 
+	if (IS_ENABLED(CONFIG_ADSP_IMR_CONTEXT_SAVE)) {
+		tuple = tlv_next(tuple);
+		tlv_value_uint32_set(tuple, IPC4_FW_CONTEXT_SAVE, 1);
+	}
```

Comment:
This has to be a negative flag, 1 if context save is not available and 0 if context save is implemented.

Reply:
sure, updated.